.github/actions/setup-prestashop-env/action.yml

```diff
@@ -27,6 +27,10 @@ inputs:
     required: false
     description: False to disable dev mode
     default: "true"
+  github-token:
```

Comment:
Since it's always provided shouldn't it be required instead? If it's optional then it should handle a fallback value, but it uses empty string not sure it will work as expected with an empty value as token

Reply:
I'm pretty sure an empty string will work the same as no token, but I didn't check 😅

Reply:
Just tried here: https://github.com/atomiix/PrestaShop/actions/runs/3845291813/jobs/6549272542 seems to work